backend/hitas/tests/apis/test_api_housing_company.py

```diff
@@ -447,6 +447,9 @@ def test__api__housing_company__create(api_client: HitasAPIClient, minimal_data:
     if minimal_data:
         data.update(
             {
+                "business_id": "",
+                "primary_loan": "",
```

Comment:
this seems to be aligned with other fields but I feel it's strange to allow empty string `""` for a numeric field :thinking: